<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="btnBoxOkCancel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editChooseCommand">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string extracomment=" "/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChooseCmd">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChooseIcon">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editChooseIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string extracomment=" "/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editChooseDecription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string extracomment=" "/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tooltip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editChooseTooltip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string extracomment=" "/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBoxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
